components/studentPortal/freemium/FreemiumDialogueComponent.tsx

```diff
@@ -20,15 +20,19 @@ export interface FreemiumDialogComponentProps {
   trigger: boolean;
   triggerTitle?: string;
   onClose: () => void;
+  startOnUpgradeModal: boolean;
 }
 
 const FreemiumDialogComponent: React.FC<FreemiumDialogComponentProps> = ({
   trigger,
   children,
   onClose,
+  startOnUpgradeModal,
 }) => {
   const { currentTheme } = useSelector(themeSelector);
-  const [activeModal, setActiveModal] = useState(ModalStage.ONE);
+  const [activeModal, setActiveModal] = useState(
+    startOnUpgradeModal === true ? ModalStage.TWO : ModalStage.ONE
```

Comment:
if you're referencing a boolean in conditional logic the default case is true so you don't need to explicitly check if it's true

this can just be startOnUpgradeModal ? ModalStage.TWO : ModalStage.ONE

Reply:
right! made the change!